### PR TITLE
Remove Test Cartridges and BIOS entries

### PR DESCRIPTION
**Pull request by RobLoach**
- **Remove Test Cartridges**
- **Remove [BIOS] entries**

### CODE REVIEW

File: dat/Nintendo - Nintendo Entertainment System.dat
Comment (steven-sheehy) [inline dat/Nintendo - Nintendo Entertainment System.dat:1]: Why are there 31K additions and deletions to this file?

**Review by steven-sheehy — COMMENTED** [review]
By my count there are 1274 entries in the RDBs without a name or rom_name. So I think we're missing a lot here. Think it would be better to keep all entries and move filtering to c_converter.

File: dat/Nintendo - Nintendo Entertainment System.dat
Comment (RobLoach) [reply dat/Nintendo - Nintendo Entertainment System.dat:1]: Possibly line endings?

**Comment by RobLoach** [comment]
Definitely always good to add improvements to c_converter. Its source is handled through the RetroArch repository, so movement can be slower there than what we can accomplish here. For now, I'll merge this, and we can do some brainstorming about ways to improve c_converter. Thanks!

**Pull request merged**